static/css/daccs-css.css

```diff
@@ -177,6 +174,15 @@
   -webkit-text-fill-color: transparent;
 }
 
+/*Page Elements*/
+#nodeContentDisplay{
```

Comment:
How are these related to this PR? I don't see them used anywhere here.

Reply:
They're not.  I just moved them from their previous location because I felt the order of the classes should more or less follow page elements.  So I moved them in between the body classes and footer.